packages/bpk-component-rating/src/BpkRating.module.scss

```
@@ -22,127 +22,55 @@ $bpk-spacing-v2: true;
 
 .bpk-rating {
   display: flex;
+  flex-flow: row nowrap;
+  align-items: baseline;
 
-  &--vertical {
-    flex-direction: column;
-  }
-
-  &__component {
-    justify-content: center;
+  &--large {
     align-items: center;
-    border-radius: 50%;
-    color: $bpk-color-white;
   }
 
-  &--base-rating {
+  &__score {
     display: flex;
-    width: bpk-spacing-lg() * 2;
-    height: bpk-spacing-lg() * 2;
-  }
+    padding-right: bpk-spacing-sm() * 2;
+    justify-content: center;
+    align-items: baseline;
+    color: $bpk-color-sky-gray;
 
-  &--lg-rating {
-    display: flex;
-    width: bpk-spacing-xxl() + (bpk-spacing-sm() * 5);
-    height: bpk-spacing-xxl() + (bpk-spacing-sm() * 5);
+    @include bpk-rtl {
+      padding-left: bpk-spacing-sm() * 2;
+    }
   }
 
-  &--sm-rating {
+  &__text-wrapper {
     display: flex;
-    width: bpk-spacing-sm() * 9;
-    height: bpk-spacing-sm() * 9;
-  }
-
-  &--sm-pill {
-    width: bpk-spacing-sm() * 9;
-    height: bpk-spacing-xl() - (bpk-spacing-sm() / 2);
-    border-radius: $bpk-border-radius-pill;
-  }
+    flex-direction: row;
+    align-items: baseline;
+    white-space: nowrap;
 
-  &--base-pill {
-    width: bpk-spacing-lg() * 2;
-    height: bpk-spacing-sm() * 9;
-    border-radius: $bpk-border-radius-pill;
+    &--large {
+      flex-direction: column;
+      align-items: flex-start;
+    }
   }
 
-  &--lg-pill {
-    width: (bpk-spacing-xl() * 2) + (bpk-spacing-sm() / 2);
-    height: bpk-spacing-xxl() + (bpk-spacing-sm() / 2);
-    border-radius: $bpk-border-radius-pill-lg;
+  &__scale {
+    color: $bpk-color-sky-gray-tint-02;
   }
 
-  &__text-wrapper {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-
-    &--horizontal {
-      // We could use `padding-inline-start: $bpk-spacing-sm` here, but it's unsupported in IE
-      padding-left: bpk-spacing-sm() * 3;
+  &__subtitle {
+    padding-left: bpk-spacing-sm() * 2;
```

As this is 8px we can use the `md` token here
```suggestion
    padding-right: bpk-spacing-md();
```